website/layouts/shortcodes/person-card.html

```
@@ -0,0 +1,19 @@
+{{- $name := .Get "name" -}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)